ci-operator/step-registry/openshift/image-based/upgrade/seed/create/periodic/openshift-image-based-upgrade-seed-create-periodic-commands.sh

```diff
@@ -59,20 +58,18 @@ export LCA_IMAGE="${LCA_PULL_REF}"
 cd ${remote_workdir}/ib-orchestrate-vm
 
 # Create the seed vm
-make seed
+make seed-vm-create wait-for-seed dnsmasq-workaround seed-cluster-prepare
 
 # Create and push the seed image
 make trigger-seed-image-create SEED_IMAGE=${SEED_IMAGE}:${SEED_IMAGE_TAG}
 
-echo "Waiting for seed creation to finish"
+echo "Waiting 10 minutes for seed creation to finish"
+sleep 10m
 until oc --kubeconfig ${seed_kubeconfig} wait --timeout 5m seedgenerator seedimage --for=condition=SeedGenCompleted=true; do \
```

Comment:
```suggestion
echo "Waiting at most 15 minutes for seed creation to finish"
until oc --kubeconfig ${seed_kubeconfig} wait --timeout 15m seedgenerator seedimage --for=condition=SeedGenCompleted=true; do \
```

Reply:
I'm confused on why you want to change the wait timings